### ISSUE TITLE
MG::Button#on_touch broken?

### ISSUE THREAD

**Issue by glennfu** [HipByte/motion-game#9]
I tried following some of the tutorial guides, and got myself a short working app with some buttons. I wanted to use the new touch events, so I upgraded to motion-game 1.1, only to find my buttons no longer working. the on_touch simply never gets called.

I tried launching SpaceGame because it's the only sample game that uses MG::Button. I found I needed to replace all instances of MC to MG to get it to build and launch, but then it instantly crashed with:

```
±  |master ✗| → rake ios:simulator debug=1
     Build build/ios/iPhoneSimulator-9.2-Development
     Build /Users/Glenn/RubymineProjects/motion-game/lib/motion-game/../../build/ios
    Create build/ios/iPhoneSimulator-9.2-Development/SpaceGame.app/Info.plist
  Simulate build/ios/iPhoneSimulator-9.2-Development/SpaceGame.app
*** Starting simulator
(lldb) command source -s 0 '/var/folders/n7/pxh84cw912q2974kd4bq6xg40000gn/T/_simgdbcmds_ios'
Executing commands in '/var/folders/n7/pxh84cw912q2974kd4bq6xg40000gn/T/_simgdbcmds_ios'.
(lldb) process attach -p 59233
Process 59233 stopped
* thread #1: tid = 0x7b136, 0x00007fff6cdd7000 dyld`_dyld_start, stop reason = signal SIGSTOP
    frame #0: 0x00007fff6cdd7000 dyld`_dyld_start
dyld`_dyld_start:
->  0x7fff6cdd7000 <+0>: popq   %rdi
    0x7fff6cdd7001 <+1>: pushq  $0x0
    0x7fff6cdd7003 <+3>: movq   %rsp, %rbp
    0x7fff6cdd7006 <+6>: andq   $-0x10, %rsp

Executable module set to "/usr/lib/dyld".
Architecture set to: x86_64h-apple-ios.
(lldb) command script import /Library/RubyMotion/lldb/lldb.py
(lldb) breakpoint set --name rb_exc_raise
Breakpoint 1: no locations (pending).
WARNING:  Unable to resolve breakpoint to any actual locations.
(lldb) breakpoint set --name malloc_error_break
Breakpoint 2: no locations (pending).
WARNING:  Unable to resolve breakpoint to any actual locations.
(lldb) continue
1 location added to breakpoint 1
1 location added to breakpoint 2
libpng warning: iCCP: known incorrect sRGB profile
libpng warning: iCCP: known incorrect sRGB profile
Process 59233 resuming
Process 59233 stopped
* thread #1: tid = 0x7b136, 0x0000000100352066 SpaceGame`cocos2d::Node::addChild(cocos2d::Node*) + 38, queue = 'com.apple.main-thread', stop reason = EXC_BAD_ACCESS (code=1, address=0x1b4)
    frame #0: 0x0000000100352066 SpaceGame`cocos2d::Node::addChild(cocos2d::Node*) + 38
SpaceGame`cocos2d::Node::addChild:
->  0x100352066 <+38>: movl   0x1b4(%rcx), %edx
    0x10035206c <+44>: movq   -0x10(%rbp), %rcx
    0x100352070 <+48>: addq   $0x1f0, %rcx
    0x100352077 <+55>: movq   %rdi, -0x18(%rbp)

(lldb)
```

I'm not sure what to try next! Let me know if I can provide any other info.

**Comment by lrz** [comment]
I'm afraid this is a bug in the latest released version of motion-game. It has been fixed in master since (as of 7d282eef3a5ad6c69ab6e986054e402fae6d5809), we will be doing a release soon but feel free to build the source if you want the fix sooner.

**Comment by glennfu** [comment]
I would love to! Unfortunately I cannot because of #10
